frontend/src/app/login/login.component.html

```diff
@@ -23,6 +23,7 @@
             </mat-card-actions>
             <mat-card-actions>
                 <a mat-button routerLink=".">Help</a>
+                <a mat-button routerLink="../privacy">Privacy</a>
```

Comment:
.. is unnecessary. `../privacy` can be reduced to `/privacy`.

Reply:
fixed route path